helloworld/asm/main.s

```diff
@@ -1,10 +1,8 @@
-	.text
-	.globl	entrypoint
+.globl entrypoint
 entrypoint:
 	lddw r1, .message
 	mov64 r2, 12
 	call sol_log_
 	exit
-	.section	.rodata
-.message:
-	.asciz	"Hello world!"
+.rodata 
+	message: .ascii "Hello world!"
```

Comment:
I could have sworn that I used `.asciz` for a reason, but now I can't remember that reason, so let's do it!